src/daidepp/grammar/grammar_utils.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, rules: str = "", **more_rules) -> None:
     def _set_try_tokens(self):
         try_tokens = self.get("try_tokens")
 
-        if try_tokens == None:
+        if try_tokens is None:
```

Comment:
linter should have caught this

Reply:
`pylint` is being installed, but it is never called.

Reply:
🤷 I expect people to run their own linters in their editors or periodically in the command line

Reply:
It's not set up in GitHub Actions and has no configuration in `setup.cfg`, and I really didn't want to try linting this large, unlinted codebase.

I think I only noticed this specific problem because PyCharm pointed it out.